main/app_mqtt.c

```diff
@@ -108,30 +108,46 @@ extern const uint8_t mqtt_iot_cipex_ro_pem_start[] asm("_binary_mqtt_iot_cipex_r
 bool handle_scheduler_mqtt_event(esp_mqtt_event_handle_t event)
 {
 #if CONFIG_MQTT_RELAYS_NB
-  if (strncmp(event->topic, SCHEDULER_CMD_TOPIC, strlen(SCHEDULER_CMD_TOPIC)) == 0) {
+  char schedulerId = get_topic_id(event, MAX_SCHEDULER_NB, SCHEDULER_CFG_TOPIC);
+
+  if (schedulerId != JSON_BAD_TOPIC_ID) {
     struct SchedulerCfgMessage s;
+    s.schedulerId = schedulerId;
 
     char tmpBuf[MAX_MQTT_DATA_SCHEDULER];
     memcpy(tmpBuf, event->data, event->data_len);
     tmpBuf[event->data_len] = 0;
     cJSON * root   = cJSON_Parse(tmpBuf);
     if (root) {
-      cJSON * schedulerTimestamp = cJSON_GetObjectItem(root,"schTst");
-      if (schedulerTimestamp) {
-        s.schedulerTimestamp = schedulerTimestamp->valueint;
+      cJSON * timestamp = cJSON_GetObjectItem(root,"ts");
```

Comment:
1. timestamp shall be not be a string?, e.g. yyyy MMdd HH:mm:ss
    some clients will not send integers as timestamps

Reply:
I would expect UNIX timrstamp